virttest/libvirt_xml/capability_xml.py

```diff
@@ -268,6 +268,21 @@ def get_cell(self):
             cell_list.append(new_cell)
         return cell_list
 
+    def get_cell_with_memory(self):
```

Comment:
suggest use old get_cell_with_memory(self) and add one parameter with default value to diff them rather than create new method

Reply:
Where is the old one ? Do you mean get_cell()?

Reply:
yes, reuse get_cell()

Reply:
Done